src/client/QXmppTrustMemoryStorage.cpp

```diff
@@ -91,6 +94,24 @@ QXmppTrustMemoryStorage::QXmppTrustMemoryStorage()
 {
 }
 
+QFuture<void> QXmppTrustMemoryStorage::setSecurityPolicies(const QString &encryption, const QXmppTrustStorage::SecurityPolicy securityPolicy)
+{
+    if (encryption.isEmpty()) {
+        d->securityPolicies.clear();
+    } else if (securityPolicy == QXmppTrustStorage::NoSecurityPolicy) {
+        d->securityPolicies.remove(encryption);
+    } else {
+        d->securityPolicies.insert(encryption, securityPolicy);
+    }
+
+    return makeReadyFuture();
+}
+
+QFuture<QXmppTrustStorage::SecurityPolicy> QXmppTrustMemoryStorage::securityPolicy(const QString &encryption)
+{
+    return makeReadyFuture(std::move(d->securityPolicies.value(encryption)));
+}
+
```

Comment:
this causes doxygen warnings (not in \cond) and it's located above the destructor?